pom.xml

```diff
@@ -172,6 +172,12 @@
       <groupId>io.spray</groupId>
       <artifactId>spray-json_2.10</artifactId>
       <version>1.3.2</version>
+     <exclusions>
```

Comment:
The new plugin gave a warning about depending on multiple Scala version.  Spray was pulling in 2.10.5 while everything else was pulling in 2.10.4.  This isn't a huge deal since minor versions are binary compatible, but for consistency I decided to exclude the 2.10.5 version.